web/src/app/components/HighlightText.tsx

```diff
@@ -34,6 +35,13 @@ export default function HighlightText({originalText, userInput, onChange, onCorr
         }); 
 
         setGameText(gameData)
+
+        setTimeout(() => {
```

Comment:
i don't love this solution since it's a completely separate from the actual countdown. If we were to change anything about how the countdown works, we would need to remember to change it here as well.

could we create inputRef in GameWindow, then pass it as a prop to HighlightText and assign it to the html input? then, when countdown === 0 in gameWindow we could set input.current.focus()

Reply:
Yeah that can work, it'll just be more coupled

Reply:
This solution also assumes that this timeout and the countdown will align, when in reality there could be rendering and network delays. which would cause a small disconnect between the countdown ending and the input focusing

Reply:
this is more work to implement, but maybe a better solution is to make the whole countdown blur thing part of highlight text

Reply:
> This solution also assumes that this timeout and the countdown will align, when in reality there could be rendering and network delays. which would cause a small disconnect between the countdown ending and the input focusing

Wait we shouldn't need the timeout, we just need the countdown. When the countdown hits 0, we just focus.

Reply:
i have a better idea, will DM you